comfy_cli/tracking.py

```
@@ -53,20 +130,38 @@ def disable():
-        mp.track(distinct_id=user_id, event_name=event_name, properties=properties)
-    except Exception as e:
-        logging.warning(f"Failed to track event: {e}")  # Log the error but do not raise
+    properties = {**properties, "cli_version": cli_version, "tracing_id": tracing_id}
+
+    for provider in PROVIDERS:
+        provider_event_name = (
+            mixpanel_name if (mixpanel_name is not None and isinstance(provider, MixpanelProvider)) else event_name
+        )
+        try:
+            provider.track(provider_event_name, distinct_id=user_id, properties=dict(properties))
+        except Exception as e:
```

Pylint (4.0.5)

[warning] Catching too general exception Exception

(W0718)